components/raftstore/src/store/read_queue.rs

```diff
@@ -25,6 +26,8 @@ where
     pub cmds: MustConsumeVec<(RaftCmdRequest, Callback<S>, Option<u64>)>,
     pub renew_lease_time: Timespec,
     pub read_index: Option<u64>,
+    pub addition_request: Option<Box<raft_cmdpb::ReadIndexRequest>>,
+    pub locked: Option<Box<LockInfo>>,
```

Comment:
Why Box here?

Reply:
It's for performance considerations.

`ReadIndexRequest` is moved frequently. `LockInfo` is a protobuf struct which is quite large. I want to reduce the size addded to `ReadIndexRequest`.